readme/BREAKING_CHANGES.markdown

```diff
@@ -802,4 +802,94 @@ be made available at
 This change provides the latest features offered by the Portlet 3.0
 Specification, which was released in early 2017.
 
+---------------------------------------
+
+### Changed the Dependency for the liferay-util:html-top JSP tag When Implementing Portlet 3.0
+- **Date:** 2018-Jun-07
+- **JIRA Ticket:** LPS-81983
+
+#### What changed?
+
+When implementing Portlet 3.0, the usage of `portal-kernel`'s `StringBundler`
+has been deprecated in favor of Petra's `StringBundler`.
```

Comment:
@codyhoag While it is true that this change was done by me during the course of Portlet 3.0 development, it is not related to Portlet 3.0. So I think it deserves it's own section, maybe "Taglib Compatibility" or something like that.

Reply:
@ngriffin7a I'll remove any mention of Portlet 3.0 since it isn't directly related. We list breaking changes as one general list. So there aren't any *sections*, just one list of breaking changes. Is it OK as it is described if I just remove the references to Portlet 3.0?

Reply:
@codyhoag I suppose it's fine if you remove the references to Portlet 3.0